optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -47,13 +48,22 @@
 from optimum.intel.generation import BaseModelForCausalLM
 
 from ...modeling_base import OptimizedModel
-from ..utils.import_utils import _torch_version, is_itrex_available, is_torch_version
+from ..utils.import_utils import (
+    _torch_version,
+    is_torch_version,
+)
 from .configuration import INCConfig
+from .quantization import weight_only_quantization
 from .utils import QUANTIZATION_CONFIG_NAME
 
 
 logger = logging.getLogger(__name__)
-
+handler = logging.StreamHandler()
+formatter = logging.Formatter("%(asctime)s [%(levelname)s] %(message)s", "%Y-%m-%d %H:%M:%S")
+handler.setFormatter(formatter)
+logger.propagate = False
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
not sure we wanna start doing this here, will create a heterogeneity between the different loggers of the lib.
maybe another PR that manages all loggers and logging messages in the lib would make more sense.

Reply:
@echarlaix wdyt ?

Reply:
please see the diff about logging " The quantized model parameters will be .........".
without logger setting
```
2024-09-05 01:41:38 [WARNING][auto_accelerator.py:422] Auto detect accelerator: CPU_Accelerator.
/home/changwa1/optimum-intel-chang/optimum/intel/neural_compressor/modeling_base.py:161: DeprecationWarning: Weight only quantization provided by intel_extension_for_transformers is deprecated and it is provided by INC now.
  warnings.warn(
The quantized model parameters will be saved in the same format as GPTQ, here is the sample model https://huggingface.co/TheBloke/Llama-2-7B-Chat-GPTQ/tree/main for details.
2024-09-05 01:41:41 [INFO][utils.py:379] Do GPTQ algorithm with config GPTQConfig {
  "local": {
    ".*lm_head": {
```
with logger setting.
```
2024-09-05 01:43:42 [WARNING][auto_accelerator.py:422] Auto detect accelerator: CPU_Accelerator.
/home/changwa1/optimum-intel-chang/optimum/intel/neural_compressor/modeling_base.py:167: DeprecationWarning: Weight only quantization provided by intel_extension_for_transformers is deprecated and it is provided by INC now.
  warnings.warn(
2024-09-05 01:43:44 [INFO] The quantized model parameters will be saved in the same format as GPTQ, here is the sample model https://huggingface.co/TheBloke/Llama-2-7B-Chat-GPTQ/tree/main for details.
2024-09-05 01:43:44 [INFO][utils.py:379] Do GPTQ algorithm with config GPTQConfig {
```

Reply:
> not sure we wanna start doing this here, will create a heterogeneity between this logger and the other loggers in the lib. maybe in another PR that adds utils to manage all loggers and logging messages in the lib would make more sense.

yes completely agree on this, can this be include in this PR @changwangss or do you prefer to do this in a following PR ?